ckcp/test.sh

```diff
@@ -35,66 +48,86 @@ get_namespace() {
   echo "$KCP_NS_NAME"
 }
 
+kcp_kubeconfig() {
+    mapfile -t files < <(find "$WORK_DIR/credentials/kubeconfig/kcp/" -name \*.kubeconfig)
+    echo "${files[0]}"
+}
+
+compute_kubeconfig() {
+    mapfile -t files < <(find "$WORK_DIR/credentials/kubeconfig/compute/" -name \*.kubeconfig)
+    echo "${files[0]}"
+}
+
+init() {
+  KUBECONFIG_KCP="$(kcp_kubeconfig)"
+  KUBECONFIG="$(compute_kubeconfig)"
+  export KUBECONFIG
+  CASES="${CASES:-"pipelines,triggers"}"
```

Comment:
currently we don't have triggers working right? 
If we default CASES to pipelines and triggers, then the script would fail in case user does not provide any values for CASES. Please default it to pipelines. 
@Roming22 can take a call on chains in his PR.

Reply:
> currently we don't have triggers working right?

right. fixed.

Reply:
Did you still not push the commit? I still see both pipeline,triggers on line 65.

Reply:
sorry for my bad. Done.